docs/resources/zapi-and-rest-gap.md

```diff
@@ -0,0 +1,7 @@
+# Zapi and Rest gaps
+
+## Volume Count difference
+`REST` and `ZAPI` collector will return different counts of volume object depending on whether you have object store servers setup on your cluster or not.
+
+- `REST` collector would return all the volumes but object store servers volumes.
+- `ZAPI` collector would return all the volumes including object store servers volumes.
```

Comment:
The REST collector does not include volume_labels for volumes associated with object store servers.

The ZAPI collector includes `volume_label`s for volumes associated with object store servers. If you have not set up any object store servers on your cluster, both collectors will return the same number of `volume_label`s.

Reply:
Done